docs/installing-metabase.md

```diff
@@ -0,0 +1,42 @@
+# Installing Metabase
+
+Welcome to Metabase! We’re excited to help you get started. We've made our install process as simple as we could to get you right to your data.
+
+Metabase is built and packaged as a Java JAR file and can be run anywhere that Java is available (after installation, Metabase is accessed via a browser).
+
+### Quick installation
+
+The quickest way to get up and running on Metabase is to [run the JAR file locally](operations-guide/running-the-metabase-jar-file.md). This is ideal when you're looking to initially test Metabase or take a new version for a spin before rolling it out to your organization.
+
+### Production installation
+
+Once you're ready to roll Metabase out across your organization, you will want to host it somewhere other than locally. If you don't have a preferred hosting service, we recommend using [Docker](operations-guide/running-metabase-on-docker.md), as it is easy to set up and maintain.
+
+If you prefer other services such as AWS Elastic Beanstalk, check out our [full list of install options](operations-guide/installing-metabase.md).
+
+## Post-install best practices
+
+Once you've successfully installed Metabase, there are a few things you can do to improve the security and stability of your instance.
+
+### Migrate your application database off of H2
+
+Metabase includes an application database. This is where Metabase stores information about users, saved questions, dashboards, etc. By default, the application database is a locally stored H2 database. We do not recommend using this H2 database in production, as it can be corrupted fairly easily. Luckily, it’s pretty simple to [migrate to a MySQL or Postgres database](operations-guide/migrating-from-h2.md).
+
+No matter the underlying application database you use, we [backing-up your database](operations-guide/backing-up-metabase-application-data.html) frequently. 
+
+### Encrypt your database credentials
+
+Further along the setup process you will need to enter credentials for your underlying databases. This information will be stored in the Metabase application database. To makes sure your information remains secure, configure Metabase to [encrypt your database details](operations-guide/encrypting-database-details-at-rest.html) at rest.
+
+### Other customization options
+
+If you’re looking for a little extra customization of your Metabase instance, we have plenty of options, ranging from setting password complexity to configuring logging. You can read all about them in our [Operations Guide](operations-guide/start.html).
+
+### Setting up Metabase
```

Comment:
it's weird that "setting up" metabase comes _after_ a bunch of fairly sophisticated choices (eg encrypting things at rest).

IMO, we should have a "hot path" of 
- Install MB on the platform of your choice
- Set up MB
- Get going and learn more about it

_then_, "here's a bunch of stuff you should probably do". We definitely want to get people from "hey I heard about this" to a chart ASAP. Anything that takes them off this path belongs in a reference manual and not a quick start path.

Reply:
For additional context, my mental model/reasoning for including some of this information is that the guides are essentially the highlight reel for each of the more detailed manuals: Install Guide -> Operations, Setting Up Metabase -> Admin, Getting Started-> Users.

That being said, I'm not sure that people on the "hot path" are even looking at the documentation. We don't put up a lot of barriers to entry to get people to their first chart. The "user journey" is:

1) Go to the marketing website
2) Click Get Started
3) See a variety of ways to install (https://metabase.com/start/)
4) Choose one
5) Get instructions on how to do set-up
6) Install and run Metabase
7) Go through the set-up wizard in the product

Looking at the numbers, the Install Guide only got around 12% of the views that the https://metabase.com/start/ page got in the last month. It also looks like the number of viewers is 5% of the number of distinct downloads. All of that makes me think people are sticking to the download journey above and not needing to deviate to the docs - which is great!

There's sort of a structural/philosophical question now - does the Install Guide actually need to be a top-tier guide on the documentation page? I could see doing something like:
1) Re-doing this page as a sort of "Advanced Installation Guide/Best Practices guide," leaving out the basic introduction to install, and linking to it from the individual installation instructions
2) Changing the copy up a bit on https://metabase.com/start/ to help people make the right choice on that page
3) Changing the install guide link to direct people to https://metabase.com/start/ OR removing it OR writing a new, third guide - maybe a general Metabase Best Practices, or a What You Need to Know Before You Start, or some other topic